server/downloadUrl.js

```diff
@@ -1,8 +1,10 @@
 var electronSettings = Meteor.settings.electron || {};
 var latestVersion = electronSettings.version;
-var downloadUrl = electronSettings.downloadUrl;
-if (downloadUrl) {
-  downloadUrl = downloadUrl.replace('{{version}}', latestVersion);
+if (electronSettings.downloadUrl){
+  var downloadUrls = {
+    "win32": electronSettings.downloadUrl.win32.replace('{{version}}', latestVersion),
+    "osx": electronSettings.downloadUrl.osx.replace('{{version}}', latestVersion)
```

Comment:
Let's use `'darwin'` instead. I have a commit on the Mixmax fork to change the app icon over to this too. I like being consistent with `process.platform`.